FrostyEditor/App.xaml.cs

```diff
@@ -29,6 +29,7 @@ public partial class App : Application
         public static PluginManager PluginManager { get => Frosty.Core.App.PluginManager; set => Frosty.Core.App.PluginManager = value; }
         public static EbxAssetEntry SelectedAsset { get => Frosty.Core.App.SelectedAsset; set => Frosty.Core.App.SelectedAsset = value; }
         public static ILogger Logger { get => Frosty.Core.App.Logger; set => Frosty.Core.App.Logger = value; }
+        public static List<int> WhitelistedBundles { get => Frosty.Core.App.WhitelistedBundles; set => Frosty.Core.App.WhitelistedBundles = value; }
```

Comment:
maybe use hashset instead

Reply:
Pretty redundant given how little this list contains but could amend anyway

Reply:
that could be a lot of bundles especially in the newer games

Reply:
Very well. Shall ammend